ejb3/src/main/java/org/jboss/as/ejb3/remote/DescriptorBasedEJBClientContextService.java

```diff
@@ -67,7 +67,7 @@ public synchronized void start(final StartContext startContext) throws StartExce
         }
         if(!profileService.getServiceUrls().isEmpty()) {
             DiscoveryProvider staticDiscoveryProvider = new StaticDiscoveryProvider(profileService.getServiceUrls());
-            contextBuilder.addDiscoveryProvider(staticDiscoveryProvider);
+            //TODO
```

Comment:
Also, please change the 'TODO' comments to 'TODO Elytron', this will help make Elytron related things easier to spot :)